Exec/RegTests/SingleDropEvap/pelelmex_prob.cpp

```diff
@@ -8,7 +8,13 @@ void
 PeleLM::readProbParm()
 {
   amrex::ParmParse pp("prob");
+
+#ifdef USE_MANIFOLD_EOS
```

Comment:
Remove `#ifdef`. Other EOS should still work with `eosparm`, which will just be a nullptr that doesn't do anything.